FuelGaugeReader/FuelGaugeReader.ino

```diff
@@ -0,0 +1,38 @@
+#define USE_USBCON
```

Comment:
What does this do?

Reply:
It keeps due from having a fit and breaking everything